### PR TITLE
Morph person to task

### PR DESCRIPTION
**Pull request by xpdavid**

### CODE REVIEW

File: src/main/java/seedu/address/model/event/EndTime.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents an Event's EndTime in the to-do List.
+ * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ */
+public class EndTime {
```
Comment (xpdavid) [inline src/main/java/seedu/address/model/event/EndTime.java]: suggestion: change the `StartTime` `EndTime` to `TimeInterval`.

**Comment by xpdavid** [comment]
Need to change the test cases.

File: src/main/java/seedu/address/model/event/EndTime.java (hunk 1 above)
Comment (haiilong) [inline src/main/java/seedu/address/model/event/EndTime.java]: Resolve in commit 6f01c50

**Comment by xpdavid** [comment]
@haiilong The class name should be named as `Task` instead of `Event`. The event is used in the 
event center. If we use `Event`, it is quite confusing.

File: .classpath [hunk 2]
```diff
@@ -17,5 +17,6 @@
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="lib" path="C:/Users/Hai Long/Downloads/joda-time-2.4/joda-time-2.4.jar"/>
```
Comment (xpdavid) [inline .classpath]: We'd better not use outside package. I don't think it is necessary to use this here.
See the project requirement.

**Pull request merged**